packages/expo-clipboard/ios/ClipboardExceptions.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+internal class InvalidImageException: Exception {
+  private let image: String
+  init(image: String) {
+    self.image = image
+  }
+  override var reason: String {
+    "Invalid base64 image: \(image.prefix(32))\(image.count ?? 0 > 32 ? "..." : "")"
+  }
+}
```

Comment:
You should rather use `GenericException` with type parameter 👀 
https://github.com/expo/expo/blob/c135332a521f3c4ec604dee8a12199ae05860d7f/packages/expo-image-manipulator/ios/ImageManipulatorExceptions.swift#L36-L40
```suggestion
internal class InvalidImageException: GenericException<String> {
  override var reason: String {
    "Invalid base64 image: \(param.prefix(32))\(param.count ?? 0 > 32 ? "..." : "")"
  }
}
```

Reply:
Thanks! That's exactly what I've been looking for 😁